deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
better use consistent naming of the filename, either "CCNP" or "ccnp", between the current "build-CCNP.sh“ and "deploy-ccnp.sh".